README.md

```diff
@@ -51,14 +51,8 @@ python3 build_tfm.py -m ARM_MUSCA_B1 -t GNUARM -c ConfigRegressionIPC.cmake
 
 ## Building the PSA Compliance Test
 
-First, run `build_psa_compliance.py` to build for a target. Different suites can
-be built using the `-s` option.
-
-```
-python3 build_psa_compliance.py -m ARM_MUSCA_B1 -s CRYPTO -t GNUARM
-```
-
-Then run `build_tfm.py` with the PSA API config.
+Run `build_tfm.py` with the PSA API config to build for a target.
```

Comment:
Could you add the following before this line

Note: If you build on _macOS_, run:
```
export SDKROOT=$(xcrun --sdk macosx --show-sdk-path)
```

Reply:
ok, please note I don't have Mac to test this. Thanks.